cmd/manager/main.go

```diff
@@ -59,6 +63,33 @@ func main() {
 
 	printVersion()
 
+	// Check if we need to watch a specific namespace
+	namespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		log.Error(err, "Failed to get watch namespace")
+		os.Exit(1)
+	}
+
+	if strings.Contains(namespace, ",") {
+		namespace = ""
+		log.Info(`This operator cannot handle multiple
+			  namespaces. The operator will watch for changes
+		          across all namespaces`)
+	}
+
+	// Set default manager options
+	options := manager.Options{
+		Namespace: namespace,
+	}
+
+	if namespace != "" && namespace != operatortypes.NsxNamespace {
```

Comment:
so if the WATCH_NAMESPACE is "nsx-system", configmap_controller won't work?

Reply:
In this case namespace value will be set to nsx-system and Options.Namespace will be nsx-system
this means all the controllers will monitor exclusively nsx-system which should be fine, as in this case the operator and operand namespace are the same